client/actions/types.js

```diff
@@ -1,10 +1,27 @@
 export const LOG_IN = 'LOG_IN';
 export const LOG_OUT = 'LOG_OUT';
+
 export const SET_RUN = 'SET_RUN';
+
+export const SET_USERS = 'SET_USERS';
+
+export const GET_SCENARIO = 'GET_SCENARIO';
+export const GET_SCENARIO_SUCCESS = 'GET_SCENARIO_SUCCESS';
+export const GET_SCENARIO_ERROR = 'GET_SCENARIO_ERROR';
+
+export const GET_SCENARIOS = 'GET_SCENARIOS';
+export const GET_SCENARIOS_SUCCESS = 'GET_SCENARIOS_SUCCESS';
+export const GET_SCENARIOS_ERROR = 'GET_SCENARIOS_ERROR';
+
 export const SET_SCENARIO = 'SET_SCENARIO';
+export const SET_SCENARIO_SUCCESS = 'SET_SCENARIO_SUCCESS';
+export const SET_SCENARIO_ERROR = 'SET_SCENARIO_ERROR';
+
 export const SET_SCENARIOS = 'SET_SCENARIOS';
+export const SET_SCENARIOS_SUCCESS = 'SET_SCENARIOS_SUCCESS';
```

Comment:
Is there going to be an api call when setting scenarios? Previously `SET_SCENARIOS` just updated the store in Redux, so there wasn't really an issue of its success or not.

Reply:
That's correct, I'm hoping to do an eventual follow up that syncs the Scenario on the server and in redux, but that seemed like a bridge too far for this PR.